src/Frontend/components/helloWorld.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
every source file (the scss files, too) should have the same header as in the old SenNetwork repository: https://github.com/Ma27/SenNetwork/blob/master/src/Sententiaregum/AppBundle/Command/PurgeNotActivatedUsersCommand.php#L3-L10